pyteal/ast/itxn.py

```diff
@@ -120,7 +120,7 @@ def Submit(cls) -> Expr:
         :any:`InnerTxnBuilder.Begin` and :any:`InnerTxnBuilder.SetField` must be called before
         submitting an inner transaction.
 
-        This will fail fail if 16 inner transactions have already been executed, or if the
+        This will fail if 16 inner transactions have already been executed, or if the
```

Comment:
I think this is not true anymore, In teal 6 I can submit up to 256 individual transactions

Reply:
@barnjamin Thanks for catching.   Can you resolve if it looks good now?

* I updated the docstring in https://github.com/algorand/pyteal/pull/193/commits/dc5289f1ea0e0368a000f775b2d230bef4379a6a.  
  * Aside:  I'm surprised docstrings _cannot_ reference variables.  I considered a larger change to make _256_ a constant and reference it with one of these methods:  https://stackoverflow.com/questions/10307696/how-to-put-a-variable-into-python-docstring.  Since there's only 1 reference, I opted to for the quick change.  Though I might circle back in a subsequent PR.
* I opened https://github.com/algorand/docs/pull/743 because I noticed a stale reference there too.